.gitignore

```
@@ -0,0 +1,2 @@
+/node_modules
+**.swp
```

Nitpick: one star is enough

```suggestion
*.swp
```